HW03.Rmd

```diff
@@ -0,0 +1,53 @@
+---
+title: "HW03"
+author: "jerry lee"
+date: "7/19/2020"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## R Markdown
+
+This is an R Markdown document. Markdown is a simple formatting syntax for authoring HTML, PDF, and MS Word documents. For more details on using R Markdown see <http://rmarkdown.rstudio.com>.
+
+When you click the **Knit** button a document will be generated that includes both content as well as the output of any embedded R code chunks within the document. You can embed an R code chunk like this:
+
+```{r cars}
+summary(cars)
+```
+
+## Including Plots
+
+You can also embed plots, for example:
+
+```{r pressure, echo=FALSE}
+plot(pressure)
+```
+
+Note that the `echo = FALSE` parameter was added to the code chunk to prevent printing of the R code that generated the plot.
+
+```{r}
+library("ggplot2")
+```
+
+```{r}
+immune<-read.csv("immune.csv", col.names=c("type","Cd3e","Cd19","Cd14"))
+```
+
+```{r}
+ggplot(immune, aes(type,Cd19,fill=type))+
+  geom_boxplot()+
+  geom_jitter(size=0.1)+
+  annotate("rect", xmin=0.5, xmax=1.5, ymin=0, ymax=15, alpha=0.2)+
+  annotate("rect", xmin=6.5, xmax=7.5, ymin=0, ymax=15, alpha=0.2)+
+  annotate("text",x=4, y=12.5, label="Cd19 seems to be a
+good marker for B cells", size=3) +
+  guides(fill=guide_legend(ncol=2)) +
+  theme(axis.text.x = element_text(angle = 90,vjust=0.5),legend.position="none")+
+  labs(title="Boxplot of relative Cd19 expression among different mouse
+samples",x="Mouse sample",y="Relative Cd19")
+```
```

Comment:
I didn't note any obvious places where you could consolidate your code. I also think it's cool that you're using your own data. I am struggling with my preview function in R Studio, so I can't actually see the graph, but from the code, I think that box plot is one way to convey your data, but I'm not sure whether it's the most straightforward. From your title, it doesn't seem like you're trying to discuss the range of a population rather just establish Cd19 as a marker of B cells. Boxplot would show this information definitely, but would anotehr format be simpler? Obviously this comment is kind of moot since I can't actually see your graph. If you were to change the format to bar or something else, I think with this type of data it would be cool to include Cd3e and/or Cd14 in the y-axis for comparison so that you can distinguish the Cd19 marker, but that might not be the message you're trying to convey. Perhaps adding both might be chaotic, but one could be meaningful. Also, I think maybe more descriptive/specific x and y labs, but this is just nit picking because I need to say three things :).